### PR TITLE
Modes of Variation Doc

### PR DESCRIPTION
**Pull request by jadie1**
Added to the "What is Shape Modeling?" section of `/docs/getting-started/shapes.md` for issue #990

### CODE REVIEW

File: docs/getting-started/shapes.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ Below are another examples of bad surfaces for shape modeling where the surface
 
 ## What is Shape Modeling?
 
+### Shape Parameters
 
 Let’s first consider parameterized geometric shapes. For example, disregarding location, orientiation, and global scale, an ellipse can be fully described by its major and minor radii. A torus can be parameterized by it s inner and outer radii.
```
Comment (sheryjoe) [inline docs/getting-started/shapes.md:48]: typo: it s -> its

**Review by sheryjoe — CHANGES_REQUESTED** [review]
@jadie1 Images are not visible.

**Comment by jadie1** [comment]
> @jadie1 Images are not visible.

They are visible for me when I run `mkdocs serve` on this branch. Are the images and videos visible for you in the rest of the documentation?

**Comment by jadie1** [comment]
@sheryjoe It seems videos on the SCI server do not show up in the documentation on github (see /docs/use-cases/ellipsoid-cutting-planes.md for another example). But they are there if you run `mkdocs serve`

**Review by akenmorris — APPROVED** [review]

**Pull request merged**